package.json

```diff
@@ -1,14 +1,18 @@
 {
-  "name": "adminr",
-  "version": "0.0.1",
-  "license": "MIT",
+  "name": "admin-as-service",
+  "version": "0.1.0",
+  "private": true,
   "dependencies": {
-    "express": "^4.15.3",
-    "express-graceful-exit": "^0.4.0",
-    "js-yaml": "^3.8.4"
+    "admin-on-rest": "^1.2.2",
+    "axios": "^0.16.2",
+    "js-yaml": "^3.9.1",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
   },
   "scripts": {
-    "start": "node index.js",
-    "example": "CONFIG_FILE=example/simple.config.yml node index.js"
```

Comment:
The example should be left in the project for preview purposes (in /examples, not in /src)